### PR TITLE
[PTX-2232] Added logic to validate applications restored by backup driver

### PR DESCRIPTION
**Pull request by vinayakshnd**

### CODE REVIEW

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -450,13 +456,13 @@ var _ = Describe("{BackupCrashVolDriver}", func() {
 			}
 		}
 
-		Step("teardown all applications on source cluster before switching context to destination cluster", func() {
+		/*Step("teardown all applications on source cluster before switching context to destination cluster", func() {
```
Comment (stgleb) [inline tests/backup/backup_test.go:459]: Why don't we tear down apps at this point?

Comment (vinayakshnd) [reply tests/backup/backup_test.go:459]: Temporarily I have disabled deletion of apps on source cluster. Will re-enable them and remove also some of the debug logs before merging.

Comment (vinayakshnd) [reply tests/backup/backup_test.go]: Uncommented with latest commit

**Comment by vinayakshnd** [comment]
1. Jenkins job run where backup test passed:
https://jenkins.portworx.co/job/tp-backup-vinayak/107/console
2. Jenkins job run, where other tests passed:
https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/489/console

**Pull request merged**